tfx/components/example_gen/big_query_example_gen/executor_test.py

```diff
@@ -64,9 +67,13 @@ def setUp(self):
     # Mock BigQuery result schema.
     self._schema = [
         bigquery.SchemaField('i', 'INTEGER', mode='REQUIRED'),
+        bigquery.SchemaField('i2', 'INTEGER', mode='REPEATED'),
```

Comment:
quick question, when you run a real BigQuery that returns an array, its scheme will looks like this?

Reply:
Yes, you can try it yourself [here](https://github.com/tensorflow/tfx/pull/1731#discussion_r421053710).